src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/VolleyUnitOfWork.cs

```diff
@@ -53,6 +53,11 @@ public Task CommitAsync()
         /// IDisposable.Dispose method implementation.
         /// </summary>
         public void Dispose()
+        {
+            Dispose(true);
+
+        }
+        protected virtual void Dispose(bool disposing)
         {
             _context.Dispose();
```

Comment:
Why don't you use declared argument in code?

Reply:
Sonar indicates that it is not properly implemented idisposible

Reply:
I can see that. Question is why don'tyou use variable you have declared?

Reply:
ok